frontend/public/components/dashboard/dashboards-page/overview-dashboard/events-card.tsx

```diff
@@ -1,38 +0,0 @@
-import * as React from 'react';
```

Comment:
Were these two files not used?

Reply:
Sorry about that, should have added comment before to explain - `events-card` is replaced by `activity-card` and `health-card` by `status-card`. The old ones are not used anywhere anymore so I removed them